projects/storefrontlib/cms-components/myaccount/my-coupons/coupon-card/coupon-card.component.html

```diff
@@ -25,11 +25,21 @@
           </div>
         </div>
       </div>
-
-      <a (click)="readMore()" class="cx-card-read-more">{{
-        'myCoupons.readMore' | cxTranslate
-      }}</a>
-
+      <a
+        *cxFeatureLevel="'!5.1'"
+        (click)="readMore()"
+        class="cx-card-read-more"
+        >{{ 'myCoupons.readMore' | cxTranslate }}</a
+      >
+      <a
```

Comment:
This should be changed to a `button` element since it does not behave like a link (ie. no navigation is done).

Reply:
Thanks, @Zeyber 
I replaced an anchor with a button to support tabbing... tested e2e and updated unit test.